script.js

```diff
@@ -1,3 +1,4 @@
+// This is a change
```

Comment:
This change is in Spanish. Can't read it.

Reply:
No, it's not. You simply can't read English.